ceph/object-storage/nano/base/kustomization.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- securitycontextconstraint.yaml
+- service-account.yaml
+- clusterrolebinding.yaml
+- persistentvolumeclaim.yaml
+- service.yaml
+- statefulset.yaml
+
+commonLabels:
+  opendatahub.io/component: "true"
+  component.opendatahub.io/name: ceph-nano
+
+configMapGenerator:
+- name: ceph-nano-parameters
+  env: params.env
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+vars:
+- name: namespace
+  objref:
+    kind: ConfigMap
+    name: ceph-nano-parameters
+    apiVersion: v1
+  fieldref:
+    fieldpath: metadata.namespace
+
+configurations:
+- params.yaml
```

Comment:
missing new line

Reply:
Missing new line where?

Reply:
after -params.yaml

Reply:
Updated.

We should add a CI job that will enforce this new style requirement for all yaml files